data/legacy/6/skills.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "type" : "skill",
+    "ident" : "carpentry",
+    "name" : "construction",
+    "description" : "Your general competence in building construction.  This governs the complexity of structures that can be built, and the time required to build them."
```

Comment:
I considered different alternatives:
* Clear carpentry on load unconditionally. But then it couldn't be re-added by a mod ever (if anyone cares)
* Clear skills that don't exist on load. Without warning it would be confusing, with warning it would mean everyone gets it when migrating
* Clear carpentry on load if it doesn't exist. This one could be an alternative here
* Just let it naturally "fade away" by obsoleting it and only keeping it around as migration thing